hashtable.js

```diff
@@ -53,6 +53,22 @@ module.exports = function (hash, matches, get) {
             })
         })(hash(key))
       },
+      getSeq: function (key, cb) {
+        ;(function next (i) {
+          var k = _get(i)
+          if(k === 0) {
+            cb(NotFound(key))
+          }
+          else
+            get(k, function (err, data) {
+              if(err) cb(err)
+              else if(matches(data, key)) {
+                cb(null, k)
```

Comment:
The `k` above is the only difference from the `get()` method -- in the `get()` method this line is instead `cb(null, data)`.